kubernetes/jobmonitor/src/main/kotlin/LostJobsFinder.kt

```diff
@@ -136,6 +160,16 @@ internal class LostJobsFinder(
         val lostJobs = jobRepository.listActive(currentTime - monitorConfig.lostJobsMinAge)
```

Comment:
So you are listing the active jobs and you are only retaining the ones that of not for this endpoint ?!

I have a hard time understanding this function and comparing it to _checkForLostSchedules_.

Maybe you should define first in its documentation what is a _lost job_ ?

Reply:
Isn't this defined in the comment of the `LostJobsFinder` class?

Reply:
Ok good enough.